src/index.js

```diff
@@ -1,31 +1,70 @@
-// import _ from 'lodash';
+import { findCity, findWeather } from './resource';
 
+const searchForm = document.querySelector('form');
+const displayCard = document.querySelector('.display-card');
+const time = document.querySelector('img.time');
+const iconImg = document.querySelector('.icon img');
+const location = document.querySelector('.location');
+const weatherText = document.querySelector('.weather-text');
+const celciusValue = document.querySelector('.cel-value');
 
-// function component() {
-//   const element = document.createElement("div");
+const addCity = async (city) => {
+  const cityKey = await findCity(city);
+  const cityWeather = await findWeather(cityKey.Key);
 
-//   // Lodash, currently included via a script, is required for this line to work
-//   element.innerHTML = _.join(["Hello", "webpack"], " ");
+  return {
+    cityKey,
+    cityWeather,
+  };
+};
+
+const convertToFahrenheit = (temp) => (temp * 9) / 5 + 32;
+
+const display = (info) => {
+  const { cityKey } = info;
+  const { cityWeather } = info;
 
-//   return element;
-// }
+  location.innerHTML = `<h5>${cityKey.EnglishName}</h5>`;
+  weatherText.innerHTML = `${cityWeather.WeatherText}`;
+  celciusValue.innerHTML = `<p>${cityWeather.Temperature.Metric.Value}&deg;C</p>`;
 
-// document.body.appendChild(component());
+  const iconSrc = `./asset/icons/${cityWeather.WeatherIcon}.svg`;
 
-const apiKey = "iiT5pz1slpmXaLAWU1cygqoLpwoAeAu3";
+  iconImg.setAttribute('src', iconSrc);
 
-const findCity = async (city) => {
-  const cityResource =
-    "http://dataservice.accuweather.com/locations/v1/cities/search";
+  const imgSrc = cityWeather.IsDayTime ? './asset/day.svg' : './asset/night.svg';
 
-  const queryKey = `?apikey=${apiKey}&q=${city}`;
+  time.setAttribute('src', imgSrc);
 
-  const response = await fetch(cityResource + queryKey);
-  const data = await response.json();
+  if (displayCard.classList.contains('d-none')) {
+    displayCard.classList.remove('d-none');
+  }
 
-  return data[0];
+  celciusValue.addEventListener('click', (e) => {
+    e.preventDefault();
+    if (cityWeather.Temperature.Unit === 'C') {
+      let fahrenheit = convertToFahrenheit(cityWeather.Temperature.Metric.Value);
+      fahrenheit = Math.floor(fahrenheit);
+
+      celciusValue.innerHTML = `<p>${fahrenheit}&deg;F</p>`;
+      cityWeather.Temperature.Unit = 'F';
+    } else {
+      celciusValue.innerHTML = `<p>${cityWeather.Temperature.Metric.Value}&deg;C</p>`;
+      cityWeather.Temperature.Unit = 'C';
+    }
+  });
```

Comment:
[OPTIONAL]

- [ ] From user's experience point of view, it'll be nice to make the toggle feature very obvious. A new user might be unaware that they could click on the temperature value to toggle between celcius and farhenhiet. A toggle button would do in this regard.

- [ ] As pointed out by the last reviewer, it'll be nice to have a button to submit the city instead of having to use the keyboard.
- [ ] Consider adding the `start` script to your package.json so that users could start the application easily by running `npm start` instead of having to launch the application from the `index.html.`